deep_quoridor/src/train_alphazero.py

```diff
@@ -183,6 +184,13 @@ def main(args):
         default=10,
         help="How many time to play against each opponent during benchmarks",
     )
+    parser.add_argument(
+        "-be",
```

Comment:
Technically in posix style command line options (which i think is pretty good) arguments with a single dash are only allowed to be one letter. This works fine in argparse though, so I don't feel too strongly about it.

Reply:
ok, I'll leave it for now since we have a few more and eventually we can fix them all